src/core/cluster.py

```diff
@@ -226,6 +232,13 @@ def unit_broker(self) -> KafkaBroker:
             substrate=self.substrate,
         )
 
+    @property
+    def kraft_unit_id(self) -> int:
```

Comment:
**todo**: this needs to be called `bootstrap_unit_id` to be consistent with models no?